### DISCUSSION TITLE
part-ii/2019-88/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#236] [Question comments]
Part II 2019 Logic and Set Theory: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2019-88/

**Comment by artie2000** [comment]
<details>
<summary>(i):</summary>

True: bookwork.

</details>
<details>
<summary>(ii):</summary>

False: $\omega_1+\omega_2=\omega_2<\omega_2+\omega_1$.
This probably isn't even the best example, but cardinalities are easy to work with.

</details>
<details>
<summary>(iii):</summary>

True: bookwork.

</details>
<details>
<summary>(iv):</summary>

False: $\omega+\omega2=\omega3=\omega2+\omega$, but $(\omega2)\omega=\omega^2<\omega^22=\omega(\omega2)$.

</details>
